src/Components/List/Card.scss

```
@@ -0,0 +1,70 @@
+@import "../../Styles/common.scss";
+@import "../../Styles/reset.scss";
+
+.Card {
+    width: 255px;
+    height: 383px;
+    margin-right: 20px;
+    display: list-item;
+    position: relative;
+    .thumnailWrap {
```

```suggestion
    position: relative;
    
    .thumnailWrap {
```
새로운 셀렉터가 시작하기 전에 한줄 띄워주세요!